### PR TITLE
Perl/web-simple - Fixed Failing tests

### PR DESCRIPTION
**Pull request by nbrady-techempower**
Fixed failing query tests and fixed all warnings.
````
================================================================================
  Verification Summary

| Test: web-simple-raw
|       query       : PASS
|       db          : PASS
================================================================================
````

### CODE REVIEW

File: frameworks/Perl/web-simple/app.pl [hunk 1]
```diff
@@ -3,7 +3,7 @@
 use JSON::XS;
 use DBI;
 
-my $dsn = "dbi:mysql:database=hello_world;host=localhost";
+my $dsn = "dbi:mysql:database=hello_world;host=127.0.0.1";
```
Comment (msmith-techempower) [inline frameworks/Perl/web-simple/app.pl]: This change seems unnecessary, but if it **is** necessary, then [this line](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Perl/web-simple/setup.sh#L3) will have to be updated accordingly.

Comment (nbrady-techempower) [inline frameworks/Perl/web-simple/app.pl]: That must have been a change made by a sed operation, as I didn't make it.

Comment (msmith-techempower) [inline frameworks/Perl/web-simple/app.pl]: Righto - let's clean this line up, then.

Comment (nbrady-techempower) [inline frameworks/Perl/web-simple/app.pl]: All set

**Comment by msmith-techempower** [comment]
All tests pass; LGTM!

**Pull request merged**